chromium_src/chrome/browser/resources/tab_search/auto_tab_groups/auto_tab_groups_page.ts

```diff
@@ -220,6 +234,11 @@ export class AutoTabGroupsPageElement extends CrLitElement {
     this.apiProxy_.openLeoGoPremiumPage()
   }
 
+  protected onEnableTabFocusClicked_() {
+    this.apiProxy_.enableTabFocus(true)
```

Comment:
if we only ever enable it from here, maybe we don't need to pass the boolean in (in which case, ignore my naming suggestion for the mojom method)